frontend/integration-tests/views/form.view.ts

```diff
@@ -0,0 +1,3 @@
+import { $ } from 'protractor';
+
+export const firstDropdownMenuByTestID = (id: string) => $(`[data-test-dropdown-menu=${id}]`);
```

Comment:
Drop the `first` prefix :)

```suggestion
export const dropdownMenuForID = (id: string) => $(`[data-test-dropdown-menu=${id}]`);
```

Reply:
fixed